pkg/server/server.go

```diff
@@ -129,9 +132,16 @@ func NewServer(ctx context.Context, cfg *config.Config, logger *zap.Logger, name
 			return
 		}
 
-		if namespaceFiles != "" {
-			err = srv.ConfigManager.ImportNamespaceFromDir(ctx, namespaceFiles)
-			if err != nil {
+		if errors.Is(dirErr, os.ErrNotExist) {
+			// first time running
+			nsc := &config.Namespace{
+				Namespace: "default",
+				Backend: config.BackendNamespace{
+					Instances:    []string{"127.0.0.1:4000"},
```

Comment:
My intention is to easily test without specifying a namespace directory.
If you insist to always create a default namespace (like a system namespace), at least don't use configurations like this. This seems like used for testing.

Reply:
Do you have any suggestion? PD and security is all configured in the config file. Backend only makes sense without setting pd. I can not come up with any better default configuration.

Reply:
You can leave it empty. Backend instances are only used for testing when there is no PD.

Reply:
Done.